releasenotes/notes/remove-forwarder_retry_queue_max_size-a4b473b1249081f4.yaml

```diff
@@ -0,0 +1,17 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    `forwarder_retry_queue_max_size` is deprecated in favor of 
```

Comment:
nit: please fix the reST format, for example: 

```suggestion
    ``forwarder_retry_queue_max_size`` is deprecated in favor of 
```

Reply:
Let's mention explicitly that `forwarder_retry_queue_payloads_max_size` takes precedence over `forwarder_retry_queue_max_size` as well